File: app/src/main/java/com/hustunique/morii/music/MusicSelectPresenter.java
package com.hustunique.morii.music;

import com.hustunique.morii.util.AudioExoPlayerUtil;

public class MusicSelectPresenter implements MusicSelectContract.IPresenter {

    private MusicSelectContract.IView view;

    MusicSelectPresenter(MusicSelectContract.IView view) {
        this.view = view;
    }


    @Override
    public void switchMusic(int position) {
        AudioExoPlayerUtil.playMusic(position);
    }

    @Override
    public void stopMusic() {
        AudioExoPlayerUtil.pauseMusicPlayer();
    }

}

File: app/src/main/java/com/hustunique/morii/music/MusicSelectPresenter.kt
package com.hustunique.morii.music

import com.hustunique.morii.util.AudioExoPlayerUtil

class MusicSelectPresenter internal constructor() :
    MusicSelectContract.IPresenter {
    override fun switchMusic(position: Int) {
        AudioExoPlayerUtil.playMusic(position)
    }

    override fun stopMusic() {
        AudioExoPlayerUtil.pauseMusicPlayer()
    }
}